modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm

```diff
@@ -559,7 +559,7 @@
 // Donation reward for inferno707
 /obj/item/storage/belt/sabre/darksabre
 	name = "ornate sheathe"
-	desc = "An ornate and rather sinister looking sabre sheathe."
+	desc = "An ornate and sabre sheath seemingly made to keep a certain ritualistic sabre safe."
```

Comment:
1. this doesnt need to be changed
2. bad grammar
3. yours sounds much more like a reference to something than the original

Reply:
its a reference to the dark saber, the item that the sheath is made for.